common/services/index.js

```diff
@@ -1,5 +1,6 @@
 const allocation = require('./allocation')
 const courtHearing = require('./court-hearing')
+const document = require('./document')
```

Comment:
Discovered it was missing while messing about. We'd need to make sure new services are added here if they stand a chance of getting the req.

Reply:
So does this mean that we need to add in all the other ones like `referenceData` and `locationsFreeSpaces`?

Reply:
Ah, good spot for locationsFreeSpaces! The reference data is already there :)